api-specificatie/openapi.yaml

```diff
@@ -198,7 +198,12 @@ paths:
           content:
             application/hal+json:
               schema:
-                $ref: '#/components/schemas/LigplaatsHAL'
+                oneOf:
```

Comment:
Het gebruiken van oneOf geeft (nog) problemen in code generatie

Reply:
Issue aangemaakt. voor nu laat ik deze nog staan om de discussie aan tw wakkeren.